org.sigxcpu.Livi.json

```diff
@@ -7,7 +7,6 @@
     "finish-args" : [
         "--device=all",
         "--share=ipc",
-        "--share=network",
```

Comment:
Network access is needed to play URLs, no?

Reply:
Hm yes - I just figured that one actually *can* easily use URLs by pasting them. Will add it back.